lib/blocs/request_manager/online_request_manager.dart

```diff
@@ -99,7 +99,7 @@ class OnlineRequestManager implements RequestManager {
       .replace(queryParameters: {
         "fromPlace": from.toString(),
         "toPlace": to.toString(),
-        "date": "01-01-2018",
+        "date": "04.03.2019",
```

Comment:
Can we make this the current date in general?

Reply:
Is this also a question for the backend team? At the beginning we had to specify this date as a fixed date because we prepared only one day in our gtfs file.

Reply:
Ahh okay, I was not aware of that.

Reply:
Don't think it's necessary though. We currently specify the timetable as valid from year 2000-2100. And if there would be a change we could control this from the backend. So the client should send the date it is searching for, which in our case would always be today for now.

Reply:
Right. We set the date to a fixed day, because the first version of our GTFS was limited to a specific date. If we you changed that to a valid timetable, it should be fine